docs/examples/example.com.yaml

```diff
@@ -8,46 +8,19 @@ spec:
       value: "example.com"
     dnsNames:
       values:
-      - "example.com"
-      - "*.example.com"
+        - "example.com"
+        - "*.example.com"
+      validations:
+        - rule: !self.contains('*')
+          message: Wildcard certificates are not allowed
     usages:
-    - "server auth"
-
+      - "server auth"
   constraints:
     privateKey:
       algorithm: RSA
       minSize: 2048
-
   selector:
     issuerRef:
       name: letsencrypt-prod
       kind: Issuer
       group: cert-manager.io
----
-kind: Role
```

Comment:
Is there a reason we remove this RBAC?

Reply:
I wrote about this in the PR description. To me, it makes little sense to repeat the repetitive RBAC in the policy examples. They are not part of the policy, but required to make the policy effective. In our setup we have granted cert-manager SA RBAC to all policies, by not setting `resourceNames`. I think that is common setup, but I might be wrong. I'm ok to leave the RBAC as is if that's a requirement to get this merged. Let me know what you think! 😄

Reply:
It makes sense. We just have to make sure we clearly document what RBAC is required (somewhere else).

Reply:
It is documented [here](https://cert-manager.io/docs/policy/approval/approver-policy/#configuration). Any documentation can be improved, but at least I could understand it when configuring our approver-policy installation.

Reply:
If we implemented https://github.com/cert-manager/approver-policy/issues/216, this wouldn't be something to discuss. 😉